examples/kibana-sample-data/docker-compose.yml

```diff
@@ -1,33 +1,33 @@
 services:
```

Comment:
Ideally, you can copy this file and create another like `examples/doris-kibana-sample-data` for your use case.